src/Sylius/Bundle/CoreBundle/Form/Type/Customer/CustomerDefaultAddressType.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\Type\Customer;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Jan Góralski <jan.goralski@lakion.com>
+ */
+class CustomerDefaultAddressType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('defaultAddress', 'sylius_address_choice', [
+                'choices_as_values' => true,
+                'customer' => $options['customerId'],
+                'choice_value' => 'id',
+            ])
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setRequired('customerId');
+        $resolver->setDefault('csrf_protection', false);
```

Comment:
It's either this or rendering a hidden token input, but I'll go with the latter.